### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by xWylo16**

### CODE REVIEW

File: InventoryGame/ammo.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package InventoryGame;
+
+import java.util.UUID;
+
+public class ammo implements item{
+    UUID id;
+    String name;
+    int amount=30;
+    int amountLimit;
+    String caliber;
```
Comment (ichi0915) [inline InventoryGame/ammo.java]: You are missing the setter and getter for the attribute caliber

File: InventoryGame/weapon.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    protected String name;
+    protected int amount;
+    protected int amountLimit;
+    protected String type;
+    protected double power;
+
+    public abstract void setType(String type);
+    public abstract String getType();
+    public abstract void setPower(double power);
+    public abstract double getPower();
```
Comment (ichi0915) [inline InventoryGame/weapon.java]: This is not abstract, implement it and remove it from Gun and Sword

File: InventoryGame/weapon.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    protected UUID id;
+    protected String name;
+    protected int amount;
+    protected int amountLimit;
+    protected String type;
+    protected double power;
+
+    public abstract void setType(String type);
+    public abstract String getType();
+    public abstract void setPower(double power);
```
Comment (ichi0915) [inline InventoryGame/weapon.java]: This is not abstract, implement it and remove it from Gun and Sword

Comment (xWylo16) [reply InventoryGame/weapon.java]: I have changed it, thanks!

**Pull request merged**